ocfweb/docs/docs/staff/procedures/ssh-supernova.md

```diff
@@ -0,0 +1,51 @@
+[[!meta title="SSHing into Supernova"]]
+
+When contributing to the OCF codebase, you will need some way of running and
+testing your code. Most of our applications cannot be run on a personal
+machine-- they need to be run on OCF infrastructure. For most projects we
+recommend doing all development from `supernova`, a server available to all
+staff. Our [[Kubernetes server|doc staff/backend/kubernetes]] is also
+accessible from `supernova`.
+
+## Before you begin
+You won't be able to log into the server if you're on staff. If you're not sure
```

Comment:
I think you might mean "if you're not on staff", but it would clearer as "You need to be on staff to log in to the server."